tests/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const URL_SEGMENT = 'http://localhost:6006/iframe.html?args=&id=';
```

Comment:
maybe these are constants and not utilities

Reply:
It is also necessary to make it so that the domain can be specified through an environment variable